consensus/service/src/config/tokens.rs

```diff
@@ -25,6 +92,12 @@ pub struct TokenConfig {
     // file.
     #[serde(default)]
     allow_any_fee: bool,
+
+    /// Master minters - if set, controls the set of keys that can sign
+    /// minting-configuration transactions.
+    /// Not supported for MOB
+    #[serde(default, with = "pem_signer_set")]
+    master_minters: Option<SignerSet<Ed25519Public>>,
```

Comment:
Preference for "governors" over "master minters," but we can discuss naming offline

Reply:
I like that name. Would prefer to do the rename in a followup PR to reduce changes to this one.